src/output-json-stats.c

```diff
@@ -204,6 +205,51 @@ static json_t *OutputStats2Json(json_t *js, const char *key)
     return value;
 }
 
+#ifdef UNITTESTS
```

Comment:
our habit (not sure if it is in the style guide) is to have the tests either at the bottom of the relevant file, or in a same-name file under src/tests/ and then include it at the bottom of this file